scripts/check_for_package_main.ts

```diff
@@ -0,0 +1,22 @@
+import { glob } from 'glob';
```

Comment:
I'd name this script `scripts/check_package_json.ts` or `scripts/check_package_jsons.ts`

If we discover a need for more custom checks like that for package.json files this will be the place to add them.